src/calliope/backend/latex_backend_model.py

```diff
@@ -362,7 +374,7 @@ def add_parameter(
         default: Any = np.nan,
         use_inf_as_na: bool = False,
     ) -> None:
-
+        """Add parameter to docs."""
```

Comment:
These (and below) shouldn't be needed. They inherit their docs from the abstract base class. By adding this line in, I expect the inherited docsstrings to be overwritten, which is not ideal as those are quite verbose. If ruff can't handle this then we need some way to deal with it that doesn't involve overwriting what's in the ABC